kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/document/datadictionary/PaymentRequestDocument.xml

```diff
@@ -12,9 +12,38 @@ xsi:schemaLocation="http://www.springframework.org/schema/beans         http://w
     <property name="attributes">
 		<list merge="true">
 			<bean parent="PaymentMethodAttribute-SelectControl"/>
+            <ref bean="PaymentRequestDocument-paymentPaidYear"/>
+            <ref bean="PaymentRequestDocument-payment1099Indicator"/>
```

Comment:
The getter for this bean originally accessed the payment1099Indicator via getPayment1099IndicatorForSearching(). The new version of Spring ignores this, so the bean has been removed and adjustments made. If you want, I can put this bean back in, making it a duplicate of the existing payment1099Indicator.